backend/src/nodes/nodes/rest/img2img.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as RESTCategory
+from ...impl.rest import (
+    decode_base64_image,
+    SamplerName,
+    STABLE_DIFFUSION_IMG2IMG_URL,
+    post_async,
+    encode_base64_image,
+)
+from ...node_base import AsyncNodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import (
+    TextInput,
+    NumberInput,
+    EnumInput,
+    ImageInput,
+)
+from ...properties.outputs import LargeImageOutput
+
+
+@NodeFactory.register("chainner:rest:sd_img2img")
+class Img2Img(AsyncNodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Modify an image using an external Stable Diffusion service"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Prompt", default="an astronaut riding a horse"),
+            TextInput("Negative Prompt", default=""),
+            NumberInput(
+                "Denoising Strength",
+                minimum=0,
+                default=0.75,
+                maximum=1,
+                controls_step=0.1,
+                precision=4,
+            ),
+            NumberInput("Seed", minimum=0, default=42, maximum=4294967296),
+            NumberInput("Steps", minimum=1, default=20, maximum=150),
+            EnumInput(SamplerName, default_value=SamplerName.EULER),
+            NumberInput(
+                "CFG Scale",
```

Comment:
Could Steps and CFG Scale also be slider inputs? The maximum values seem kinda weird, so I don't know if it makes sense.

Also, we also support sliders with log scales. So if you need that, we have it :)

Reply:
They are sliders in the Automatic1111 UI, so it would make sense here too.  The upper bounds are, I think, arbitrary but reasonable, and the same as what are in the other UI.